nginx.conf

```diff
@@ -37,7 +37,7 @@ http {
    add_header Pragma "no-cache";
 
    # Content Security Policy
-   add_header Content-Security-Policy "default-src 'self'; frame-src 'self' *.gov.bc.ca *.hotjar.com *.googleapis.com https://*.nr-data.net https://*.newrelic.com https://*.cac1.pure.cloud; script-src 'self' 'unsafe-eval' 'unsafe-inline' *.gov.bc.ca *.hotjar.com *.googleapis.com https://*.nr-data.net https://*.newrelic.com https://*.cac1.pure.cloud; style-src 'self' 'unsafe-inline' *.cloudflare.com; font-src 'self' *.gov.bc.ca *.hotjar.com *.cloudflare.com; img-src 'self' data: *.hotjar.com https://*.cac1.pure.cloud; connect-src 'self' *.gov.bc.ca *.launchdarkly.com *.hotjar.com *.postescanada-canadapost.ca *.sentry.io wss://*.hotjar.com *.hotjar.io https://*.nr-data.net https://shyrka-prod-cac1.s3.ca-central-1.amazonaws.com https://*.newrelic.com https://*.cac1.pure.cloud wss://*.cac1.pure.cloud; manifest-src 'self'; media-src 'self' https://*.cac1.pure.cloud; object-src 'self' https://*.cac1.pure.cloud; child-src 'self' https://*.cac1.pure.cloud;";
+   add_header Content-Security-Policy "default-src 'self'; frame-src 'self' *.gov.bc.ca *.hotjar.com *.googleapis.com https://*.nr-data.net https://*.newrelic.com https://*.cac1.pure.cloud; script-src 'self' 'unsafe-eval' 'unsafe-inline' *.gov.bc.ca *.hotjar.com *.googleapis.com https://*.nr-data.net https://*.newrelic.com https://*.cac1.pure.cloud; style-src 'self' 'unsafe-inline' *.cloudflare.com; font-src 'self' *.gov.bc.ca *.hotjar.com *.cloudflare.com; img-src 'self' *.hotjar.com *.postescanada-canadapost.ca https://*.cac1.pure.cloud data:; connect-src 'self' *.gov.bc.ca *.launchdarkly.com *.hotjar.com *.postescanada-canadapost.ca *.sentry.io *.apigee.net wss://*.hotjar.com *.hotjar.io https://*.nr-data.net https://shyrka-prod-cac1.s3.ca-central-1.amazonaws.com https://*.newrelic.com https://*.cac1.pure.cloud wss://*.cac1.pure.cloud; manifest-src 'self'; media-src 'self' https://*.cac1.pure.cloud; object-src 'self' https://*.cac1.pure.cloud; child-src 'self' https://*.cac1.pure.cloud;";
```

Comment:
Also this diff doesn't look like https://github.com/bcgov/business-create-ui/pull/527/files ...?

Reply:
> As for "cac1.pure.cloud", it's the Maximus security header that they gave us. It was added to many sources. As to why, I don't have a definitive answer since it's their code.

Did they ask us to add that domain to all those sources?

Reply:
So I pulled up the email they sent us:
![script](https://user-images.githubusercontent.com/122301442/225974093-2ec8b6e8-606f-4310-b282-09f9ac6d6070.PNG)

If you inspect the meta security tag, you can see all those sources added. I added all these sources to their respective positions in our header tag. They didn't directly ask us but it's there in the email they sent us. 

As for the difference, the part where I added postescanada-canadapost.ca, I just added this domain while moving "Data:" to the end of "img-src" since it looks cleaner. There's no effect of that.
So this PR only adds "apigee.net" and "postescanada-canadapost.ca" again since I omitted them by mistake.

Reply:
OK, great, thanks for all that. It matches what I'm seeing in the diff.